taf/oll/tuf_utils.py

```diff
@@ -0,0 +1,221 @@
+import getpass
+import json
+import os
+import shutil
+from contextlib import contextmanager
+from pathlib import Path
+import securesystemslib
+
+role_keys_cache = {}
+
+class TUFRepository:
```

Comment:
Maybe it makes more sense to call it `TAFRepository`?

Reply:
I am not sure which name is better. This class is meant to be a wrapper around TUF repository

Reply:
For me it's little bit confusing.
I am just thinking of instantiating this class in following way:

```python
repository = TUFRepository()
tuf_repo = repository.repository # which is `tuf.repository_tool.Repository`
```

Does it makes more sense to subclass `tuf.repository_tool.Repository`?

Reply:
The problem is that `tuf.repository_tool.Repository` is instantiated by TUF, using `load_repository`. While I agree that it would be nice to subclass it, I don't know how to do that elegantly. Do you have any suggestions?

Reply:
We also use `taf.tuf_utils.load_repository` to instantiate `taf.tuf_utils.TUFRepository`, right?
I guess it should work the same way if you just subclass `tuf.repository_tool.Repository` and just use `self.SOME_METHOD_FROM_SUBCLASS` instead of `self.repository.SOME_METHOD_FROM_SUBCLASS`?

Since we are just extending a TUF that way, we can maybe use the same module and class names.

```python
import taf.repository_tool.load_repository

with load_repository() as repo:
   repo...
```